homeassistant/components/streamlabswater/sensor.py

```diff
@@ -27,7 +27,7 @@ async def async_setup_entry(
 
     entities = []
 
-    for location_id in coordinator.data.values():
+    for location_id in coordinator.data:
```

Comment:
coordinator returns a dict it seems so shouldn't this be
```suggestion
    for location_id in coordinator.data.keys():
```